hazelcast/src/main/java/com/hazelcast/map/impl/record/AbstractRecord.java

```diff
@@ -106,10 +105,7 @@ public void setHits(long hits) {
     public long getCost() {
         final int objectReferenceInBytes = 4;
```

Comment:
This depends on JVM settings. 4 bytes is compressed oops, so if non compressed oops are used, the cost is incorrect.